lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_other_countries.govspeak.erb

```diff
@@ -10,7 +10,5 @@
                locals: {
                  overseas_passports_embassies: calculator.overseas_passports_embassies
                } %>
-  <% else %>
-    <% raise "The outcome for #{calculator.ceremony_country} is not handled" %>
```

Comment:
I wonder whether the removal of this means we can revert 50ab9b8d4963530eae7142f230d07ae75d04fb93 thus simplifying the code. If this is true, it's probably worth capturing that as a new Trello card.